portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase

```diff
@@ -949,4 +949,116 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-129536. View the configuration of widgets on content page persist after execute Clean Up Orphaned Theme Portlet Preferences."
+	@priority = "4"
+	test ViewWidgetsConfigurationOnContentPagePersistAfterCleanUpOrphanedTheme {
+		property test.name.skip.portal.instance = "ContentPages#ViewWidgetsConfigurationOnContentPagePersistAfterCleanUpOrphanedTheme";
+
+		task ("Add a navigation menu") {
+			JSONSitenavigation.addSiteNavigationMenu(
+				groupName = "Test Site Name",
+				siteNavigationMenuName = "Navigation Menu Name");
+		}
+
+		task ("Navigate to the navigation menu") {
+			NavigationMenusAdmin.openNavigationMenusAdmin(siteURLKey = "test-site-name");
+
+			LexiconEntry.gotoEntry(rowEntry = "Navigation Menu Name");
+		}
+
+		task ("Add an URL type item to navigation menu") {
+			NavigationMenusAdmin.addItem(
+				item = "URL",
+				url = "https://www.liferay.com/",
+				urlName = "Liferay");
+		}
```

Comment:
Unneeded

Reply:
> Hi @ozysouza, Could you update the cases according to comments?

Hi @Tim-Cao, Of course, I will update the pull request. 
FYI - I added some extra steps because I saw a Benjamin's comment about the Nagivation Menu.